fix(hasura): nested sorting abilities (#5508)
refinedev/refine: .changeset/hungry-readers-begin.md

---

> "@refinedev/your-package": patch

Reviewer: ```suggestion
"@refinedev/hasura": patch
```

---

> fix(utils): handle nested fields in generateSorting
> 
> This changeset addresses an issue with nested fields in the generateSorting utility function. The fix ensures that sorting works correctly when dealing with nested fields in the GraphQL query generation.
> 
> Resolves #5348

Reviewer: ```suggestion
fix(utils): handle nested fields in generateSorting #5348

This change addresses an issue with nested fields in the generateSorting utility function. The fix ensures that sorting works correctly when dealing with nested fields in the GraphQL query generation.
```

> fix(utils): handle nested fields in generateSorting
> 
> This changeset addresses an issue with nested fields in the generateSorting utility function. The fix ensures that sorting works correctly when dealing with nested fields in the GraphQL query generation.
> 
> Resolves #5348

Reviewer 2: ```suggestion
fix(utils): handle nested fields in generateSorting #5348

This changeset addresses an issue with nested fields in the generateSorting utility function. The fix ensures that sorting works correctly when dealing with nested fields in the GraphQL query generation.
```
